tardis/visualization/tools/sdec_plot.py

```diff
@@ -1105,6 +1106,11 @@ def generate_plot_mpl(
         distance : astropy.Quantity or None, optional
             Distance used to calculate flux instead of luminosity in the plot.
             It should have a length unit like m, Mpc, etc. Default value is None
+        observed_spectrum : tuple or list of astropy.Quantity, optional
+            Option to plot an observed spectrum in the SDEC plot. If given, the first element
+            should be the wavelength and the second element should be flux,
+            i.e. (wavelength, flux). The assumed units for wavelength and flux are
+            angstroms and erg/(angstroms * s * /cm^2), respectively. Default value is None.
```

Comment:
remove the / before cm^2

Reply:
done in 835477b00f91efcfd446a157bb8ca26218a4a637